### ISSUE TITLE
Cross origin error

### ISSUE THREAD

**Issue by vdidushko** [elasticio/webhook-component#33]
# Bug Report

## Description

After executing the following query in browser's console 
```const request = {
    cache: 'no-cache',
    credentials: 'include',
    headers: {
      'Authorization': 'Basic XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
    },
    method: 'get',
    mode: 'cors',
    redirect: 'follow',
    referrer: 'no-referrer'
  };

  fetch('https://in.elastic.io/hook/5ea6a4c55d7ddd5e75eb4191', request);
  ```
it is throwing the following error message:

```
Access to fetch at 'https://in.elastic.io/hook/5ea6a4c55d7ddd5e75eb4191' from origin 'https://app.elastic.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

in.elastic.io/hook/5ea6a4c55d7ddd5e75eb4191:1 Failed to load resource: net::ERR_FAILED
5b62c919fd98ea00112d52e7#/w/58dcfe0f52f01e0019e7484e:1 Uncaught (in promise) TypeError: Failed to fetch
```
![screenshot_67](https://user-images.githubusercontent.com/48764578/80379949-7a3dae00-88a7-11ea-8226-0db4339db11a.png)

# Preconditions

1. Create a new flow.
2. Add Webhook component as trigger
3. Add http reply component
4. Add custom headers to http reply component
```Access-Control-Allow-Origin: "*"```
```Access-Control-Allow-Methods: "GET"```
5. Turn on 'real time' flow
6. Start the flow

## Environment

* [ ] Great Moraq (development)
* [ ] Essos (stage)
* [X] Westeros (production)

## Steps to Reproduce

1. Run the created flow.
2. Execute early mentioned query in browser's console with the authorization header as in Webhook step.
3. Observe the error message.

## Actual Result

```
Access to fetch at 'https://in.elastic.io/hook/5ea6a4c55d7ddd5e75eb4191' from origin 'https://app.elastic.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

in.elastic.io/hook/5ea6a4c55d7ddd5e75eb4191:1 Failed to load resource: net::ERR_FAILED
5b62c919fd98ea00112d52e7#/w/58dcfe0f52f01e0019e7484e:1 Uncaught (in promise) TypeError: Failed to fetch
```

## Expected Result

Status code 200 OPTIONS

**Comment by vdidushko** [comment]
Most likely can be reproduced with any authorization method.

**Comment by jhorbulyk** [comment]
Closing in favor of https://eio.productboard.com/feature-board/2273350-planning-board/features/8357891/detail
